src/DotVVM.Framework/Routing/RouteHelper.cs

```diff
@@ -29,33 +29,6 @@ private static IEnumerable<string> GetRoutesForFile(string fileName)
             else yield return pureName;
         }
 
-        [Obsolete("Use IRoutingStrategy.")] // TODO: make this a helper for IRoutingStrategy
```

Comment:
Is the TODO comment fulfilled?

Reply:
Its behavior is similar to `DefaultRouteStrategy ` isn't it?

Reply:
Yes it is, but there should be an extension method that will apply the DefaultRouteStrategy

Reply:
So... we need to update the extension method to use `DefaultRouteStrategy`. And the route strategy must accept the `pattern` parameter.